### PR TITLE
Updated Pluging to Mautic 3

### PR DESCRIPTION
**Pull request by luguenth**

### CODE REVIEW

File: Config/config.php [hunk 1]
```diff
@@ -41,6 +42,29 @@
     ],
 
     'services' => [
+        'integrations' => [
+            'mautic.integration.triggerdialog' => [
+                'class'     => TriggerdialogIntegration::class,
+                'arguments' => [
```
Comment (flossels) [inline Config/config.php:48]: Are all these arguments really necessary?

**Review by flossels — CHANGES_REQUESTED** [review]
Hi,

please use proper file permissions and do not change it from 644 to 755.

File: Config/config.php (hunk 1 above)
Comment (luguenth) [reply Config/config.php]: no, I removed the whole integration config, which was unnecessary

File: Service/TriggerdialogService.php [hunk 2]
```diff
@@ -137,7 +137,7 @@ public function updateCampaignVariable(TriggerCampaign $triggerCampaign)
         $xml = new \SimpleXMLElement('<updateCampaignVariableRequest xmlns:ns2="urn:pep-dpdhl-com:triggerdialog/campaign/v_10"></updateCampaignVariableRequest>');
         $this->transformData($xml, $data);
 
-        $response = $this->client->request('POST', '/rest-mas/campaign/variable/', ['body' => $xml->asXML()]);
+        $response = $this->client->request('POST', '/rest-mas/variabledefinitions/', ['body' => $xml->asXML()]);
```
Comment (flossels) [inline Service/TriggerdialogService.php:140]: Is this change breaking?
Anyhow, you should consider to update the version number of this plugin.

**Review by flossels — CHANGES_REQUESTED** [review]
The issue with the file permissions still exists.

File: Service/TriggerdialogService.php (hunk 2 above)
Comment (luguenth) [reply Service/TriggerdialogService.php:140]: they changed the two route in the old api, so this will be necessary even for old clients

**Review by flossels — APPROVED** [review]

**Pull request merged**